.pre-commit-config.yaml

```diff
@@ -31,11 +31,3 @@ repos:
         language: system
         files: xrpl/
         types: [python]
-      - id: sphinx
-        name: sphinx
-        entry: poetry run sphinx-apidoc -o docs/source xrpl
-        language: system
-        pass_filenames: false
-        files: __init__
-        exclude: ^tests
-        types: [python]
```

Comment:
this seems like it was doing more harm than good

Reply:
How so?